src/state/experiments/selectors.js

```diff
@@ -1,2 +1,41 @@
 export const getExperimentsCount = state =>
   state.get("experiments").get("items").size;
+
+export const getFilteredExperimentsByDate = (state) => {
+  const experiments = state.get("experiments").get("items"); 
+  const selectedStartTimestamp = getStartDatepickerTimestamp(state);
+  const selectedEndTimestamp = getEndDatepickerTimestamp(state);
+
+  const filteredExperiments = experiments.filter(experiment => {
+    const startDate = experiment.get("start_date");
+    const endDate = experiment.get("end_date");
+
+    if (selectedStartTimestamp) {
```

Comment:
Yes exactly, this is great.  This way we can continue to add filters without needing to modify the existing ones ;D